File: JAVA LEC/L15/P2.java
//WAP to read list of marks and find sum,avg,max and min

import java.util.*;
import java.io.*;
class P2
{
public static void main(String args[])
{
Console c=System.console();
ArrayList<Integer> data=new ArrayList();

String ans=c.readLine("do u want to add element yes/no ");
while(ans.equals("yes"))
{
int d=Integer.parseInt(c.readLine("enter the data "));
data.add(d);
ans=c.readLine("do u want to add more elements yes/no ");
}
System.out.println("elements = "+data);

int sum=0;
for(Integer i:data)
sum=sum+i;
double avg=(double)sum/data.size();
Collections.sort(data);
System.out.println("elements = "+data);

int min = data.get(0);
int max = data.get(data.size() - 1);
System.out.println("sum = "+ sum + " avg = " +avg);
System.out.println("min = "+ min + " avg = " +avg);

//System.out.println("elements = "+data);

}
}

File: Python/python/L5/p2.py
#wapp to read int if marks and find sum avg min max

marks=[]
ans=input("do u want to enter marks y/n ")
while ans =='y':
	ele=int(input("enter marks "))
	marks.append(ele)
	ans=input("do u want to enter marks y/n ")
sum=0
for m in marks:
	sum=sum+m
avg=sum/len(marks)

marks.sort()
min=marks[0]
max=marks[-1]
print("sum= ",sum)
print("avg= ",avg)
print("min= ",min)
print("max= ",max)